# Translations template for nova.
# Copyright (C) 2015 ORGANIZATION
# This file is distributed under the same license as the nova project.
#
# Translators:
# Shuwen SUN <chris-void@outlook.com>, 2014
# Shuwen SUN <chris-void@outlook.com>, 2014
# 颜海峰 <yanheven@gmail.com>, 2014
# Andreas Jaeger <jaegerandi@gmail.com>, 2016. #zanata
msgid ""
msgstr ""
"Project-Id-Version: nova 14.0.0.0b3.dev750\n"
"Report-Msgid-Bugs-To: https://bugs.launchpad.net/openstack-i18n/\n"
"POT-Creation-Date: 2016-08-25 06:14+0000\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"PO-Revision-Date: 2015-10-07 05:40+0000\n"
"Last-Translator: liujunpeng <liujunpeng@inspur.com>\n"
"Language: zh-CN\n"
"Plural-Forms: nplurals=1; plural=0;\n"
"Generated-By: Babel 2.0\n"
"X-Generator: Zanata 3.7.3\n"
"Language-Team: Chinese (China)\n"

#, python-format
msgid ""
"%(cell)s has not been seen since %(last_seen)s and is being treated as mute."
msgstr "自从%(last_seen)s ，单元%(cell)s 还没有被发现，单元被认为是沉默了。"

#, python-format
msgid "%(cmd)s failed. Sleeping %(time)s seconds before retry."
msgstr "%(cmd)s 失败。重试之前，等待%(time)s  秒。"

#, python-format
msgid "%(host_state)s has not been heard from in a while"
msgstr "已经有一段时间没有收到主机状态 %(host_state)s "

#, python-format
msgid "%(logprefix)s failed to load json"
msgstr "%(logprefix)s加载 json失败"

#, python-format
msgid "%(logprefix)s file does not exist"
msgstr "%(logprefix)s 文件不存在"

#, python-format
msgid "%(logprefix)s unexpected IOError when reading"
msgstr "读取时，%(logprefix)s 未预期的IOError"

#, python-format
msgid "%(overs)s quota exceeded for %(pid)s, tried to resize instance."
msgstr "对于 %(pid)s，已超过 %(overs)s 配额，已尝试调整实例大小。"

#, python-format
msgid "%(plugin)s.%(fn)s failed. Retrying call."
msgstr "%(plugin)s.%(fn)s 失败，重试调用。"

#, python-format
msgid "%s failed. Not Retrying."
msgstr "%s 失败。不重试"

#, python-format
msgid "%s is already mounted"
msgstr "已安装 %s"

#, python-format
msgid "%s swap image was used by instance but no back files existing!"
msgstr "%s 交换镜像被实例使用，但是没有备份文件存在！"

#, python-format
msgid "Address |%(address)s| is not allocated"
msgstr "地址 |%(address)s| 没有分配"

#, python-format
msgid "Address |%(address)s| is not allocated to your project |%(project)s|"
msgstr "地址 |%(address)s| 没有分配给你的项目 |%(project)s|"

#, python-format
msgid ""
"Assigning a pci device without numa affinity toinstance %(instance)s which "
"has numa topology"
msgstr "分配 一个没有numa亲和性的pci设备给实例%(instance)s，这个实例有numa拓扑"

#, python-format
msgid "Attempted auto_configure_disk failed because: %s"
msgstr "尝试自动配置磁盘auto_configure_disk 失败，原因是：%s"

#, python-format
msgid "Attempted to remove chain %s which does not exist"
msgstr "试图移除不存在的链 %s。"

#, python-format
msgid ""
"Cannot add security group %(name)s to %(instance)s since the port "
"%(port_id)s does not meet security requirements"
msgstr ""
"无法将安全组 %(name)s 添加至 %(instance)s，因为端口 %(port_id)s 没有满足安全"
"性要求"

#, python-format
msgid "Cannot decode JSON from %(id_path)s"
msgstr "不能从 %(id_path)s 解码JSON"

#, python-format
msgid "Cannot delete domain |%s|"
msgstr "无法删除域 |%s|"

#, python-format
msgid "Cannot delete entry |%s|"
msgstr "无法删除条目 |%s|"

msgid "Cannot destroy instance, operation time out"
msgstr "不能销毁实例，操作超时"

msgid ""
"Cannot get the number of cpu, because this function is not implemented for "
"this platform. "
msgstr "无法获取 cpu 数，因为对于此平台，未实现此功能。"

msgid "Cannot purge SR with referenced VDIs"
msgstr "无法清除具有所引用 VDI 的 SR"

#, python-format
msgid ""
"Cannot update service status on host \"%s\" due to an unexpected exception."
msgstr "在主机\"%s\"上，不能更新服务状态，原因是有一个未预期的异常。"

#, python-format
msgid "Cannot update service status on host \"%s\" since it is not registered."
msgstr "在主机\"%s\"上，不能更新服务状态，因为它没有注册。"

#, python-format
msgid "Change will make usage less than 0 for the following resources: %s"
msgstr "对于下列资源：%s，更改将导致使用量小于0"

msgid ""
"Cinder V1 API is deprecated as of the Juno release, and Nova is still "
"configured to use it. Enable the V2 API in Cinder and set cinder."
"catalog_info in nova.conf to use it."
msgstr ""
"随着Juno的发布，Cinder V1 API已经被废弃，仍然配置Nova使用它。在Cinder中启用"
"V2 API，并且在nova.conf中设置使用它。"

#, python-format
msgid "Could not decode cpu_allocation_ratio: '%s'"
msgstr "不能解码 cpu 分配比率 cpu_allocation_ratio: '%s'"

#, python-format
msgid "Could not decode disk_allocation_ratio: '%s'"
msgstr "不能解码 磁盘分配比率 disk_allocation_ratio: '%s'"

#, python-format
msgid "Could not decode max_instances_per_host: '%s'"
msgstr "不能解码每台主机最大实例数max_instances_per_host: '%s'"

#, python-format
msgid "Could not decode max_io_ops_per_host: '%s'"
msgstr "不能解码每台主机的最大每秒 io 操作 max_io_ops_per_host: '%s'"

#, python-format
msgid "Could not decode ram_allocation_ratio: '%s'"
msgstr "不能解码内存分配比率 ram_allocation_ratio: '%s'"

msgid "Could not determine iscsi initiator name"
msgstr "无法确定iscsi initiator名字"

#, python-format
msgid "Could not determine key: %s"
msgstr "未能确定键：%s"

#, python-format
msgid ""
"Couldn't retrieve the online CPUs due to a Libvirt error: %(error)s with "
"error code: %(error_code)s"
msgstr "不能获取在线CPU，原因是Libvirt错误：%(error)s，错误号：%(error_code)s"

#, python-format
msgid ""
"Database inconsistency: DNS domain |%s| is registered in the Nova db but not "
"visible to either the floating or instance DNS driver. It will be ignored."
msgstr ""
"数据库不一致：DNS域|%s| 在Nova数据库中注册，但是对浮动或者实例DNS驱动均不可"
"见。将被忽略。"

#, python-format
msgid ""
"Destination %s already exists! Concurrent moves can lead to unexpected "
"results."
msgstr "目的 %s 已经存在！并发移动可能导致未预期结果。"

#, python-format
msgid "Destroy called on non-existing instance %s."
msgstr "已对不存在的实例 %s 调用“销毁”操作。"

#, python-format
msgid "Detach interface failed, port_id=%(port_id)s, reason: %(msg)s"
msgstr "解绑接口失败，port_id=%(port_id)s，原因：%(msg)s"

#, python-format
msgid "Detaching from erroneous nbd device returned error: %s"
msgstr "从不正确的 nbd 设备断开连接返回了错误：%s"

msgid "Detaching volume from unknown instance"
msgstr "正在从未知实例断开与卷的连接"

#, python-format
msgid ""
"Detected instance with name label '%s' which is marked as DELETED but still "
"present on host."
msgstr "删除标签 '%s'的实例，它被标记为删除，但是仍然在主机上。"

#, python-format
msgid ""
"Detected running orphan instance: %(uuid)s (consuming %(memory_mb)s MB "
"memory)"
msgstr "检测到孤儿实例：%(uuid)s (使用内存 %(memory_mb)s MB)"

msgid "Device allocation failed after repeated retries."
msgstr "在反复重试之后，设备分配失败。"

#, python-format
msgid "Domain |%(domain)s| already exists, changing project to |%(project)s|."
msgstr "域 |%(domain)s| 已经存在，把项目改变为 |%(project)s|。"

#, python-format
msgid "Domain |%(domain)s| already exists, changing zone to |%(av_zone)s|."
msgstr "域 |%(domain)s| 已经存在，把区域改变为 |%(av_zone)s|。"

#, python-format
msgid "Driver failed to detach volume %(volume_id)s at %(mount_point)s."
msgstr "驱动在挂载点 %(mount_point)s 解绑卷%(volume_id)s 失败。"

msgid "During detach_interface, instance disappeared."
msgstr "在 detach_interface 期间，实例消失。"

msgid "During detach_volume, instance disappeared."
msgstr "在 detach_volume 期间，实例消失。"

#, python-format
msgid ""
"Error from libvirt during destroy. Code=%(errcode)s Error=%(e)s; attempt "
"%(attempt)d of 3"
msgstr ""
"销毁期间，libvrit 报错。代号 =%(errcode)s 错误=%(e)s; 尝试 %(attempt)d /3"

#, python-format
msgid ""
"Error from libvirt while getting description of %(instance_name)s: [Error "
"Code %(error_code)s] %(ex)s"
msgstr ""
"获取 %(instance_name)s 的描述时 libvirt 中出错：[错误代码 %(error_code)s] "
"%(ex)s"

#, python-format
msgid "Error monitoring migration: %(ex)s"
msgstr "监控迁移错误：%(ex)s"

#, python-format
msgid "Error while trying to clean up image %s"
msgstr "尝试清除镜像%s时出错"

#, python-format
msgid ""
"Excluding %(namespace)s monitor %(monitor_name)s. Already loaded "
"%(loaded_monitor)s."
msgstr ""
"不包括%(namespace)s 监测器 %(monitor_name)s。已经加载 %(loaded_monitor)s。"

#, python-format
msgid ""
"Excluding %(namespace)s monitor %(monitor_name)s. Not in the list of enabled "
"monitors (CONF.compute_monitors)."
msgstr ""
"不包括%(namespace)s 监测器 %(monitor_name)s。没有在启用监测器(CONF."
"compute_monitors) 列表。"

#, python-format
msgid "Expected to receive %(exp)s bytes, but actually %(act)s"
msgstr "期望接收 %(exp)s 字节，但实际接收 %(act)s"

#, python-format
msgid ""
"Extension %(ext_name)s: Cannot extend resource %(collection)s: No such "
"resource"
msgstr "扩展%(ext_name)s：无法扩展资源 %(collection)s：没有那种资源"

#, python-format
msgid "Failed to %(service)s_%(method)s: %(ex)s"
msgstr "%(service)s_%(method)s 失败：%(ex)s"

#, python-format
msgid "Failed to abort migration %s"
msgstr "放弃 %s 迁移失败"

#, python-format
msgid "Failed to close augeas %s"
msgstr "未能关闭 augeas %s"

#, python-format
msgid "Failed to close guest handle %s"
msgstr "未能关闭 guest 句柄 %s"

msgid ""
"Failed to create block device for instance due to being over volume resource "
"quota"
msgstr "实例的块设备创建失败，原因是超过了卷资源配额"

#, python-format
msgid "Failed to deallocate port %(port_id)s for instance. Error: %(error)s"
msgstr "为实例取消分配端口%(port_id)s失败。错误：%(error)s"

#, python-format
msgid "Failed to delete port %s for instance."
msgstr "删除实例的端口 %s 失败."

#, python-format
msgid "Failed to delete snapshot from shelved instance (%s)."
msgstr "从搁浅的实例 (%s)删除快照失败。"

#, python-format
msgid "Failed to delete volume: %(volume_id)s due to %(exc)s"
msgstr "由于 %(exc)s 删除卷 %(volume_id)s 失败"

#, python-format
msgid "Failed to disassociated floating address: %s"
msgstr "浮动地址解除绑定失败：%s"

#, python-format
msgid "Failed to extract instance support from %s"
msgstr "未能从 %s 抽取实例支持"

#, python-format
msgid "Failed to get cluster references %s"
msgstr "获取集群引用 %s 失败"

#, python-format
msgid ""
"Failed to kill a long running process %(pid)s related to the instance when "
"deleting it."
msgstr "当删除实例时，杀死一个与它有关的长时间运行的进程%(pid)s失败。"

#, python-format
msgid "Failed to load extension %(ext_factory)s: %(exc)s"
msgstr "加载扩展 %(ext_factory)s 失败：%(exc)s"

msgid "Failed to power off instance"
msgstr "关闭实例失败"

msgid "Failed to resume instance"
msgstr "未能恢复实例"

#, python-format
msgid "Failed to revert task state for instance. Error: %s"
msgstr "还原实例的任务状态失败。错误：%s"

#, python-format
msgid "Failed to shutdown appliance %s"
msgstr "未能关闭设备 %s"

msgid "Failed to soft reboot instance. Trying hard reboot."
msgstr "软重启实例失败。尝试硬重启。"

#, python-format
msgid ""
"Fast cloning is only supported on default local SR of type ext. SR on this "
"system was found to be of type %s. Ignoring the cow flag."
msgstr ""
"快速克隆只在缺省的本地ext类型的SR上面支持。在这个系统上面，发现SR是类型 %s。"
"忽略cow标记。"

msgid "File injection into a boot from volume instance is not supported"
msgstr "不支持向从卷启动的实例进行文件注入"

#, python-format
msgid "Flag sr_matching_filter '%s' does not respect formatting convention"
msgstr "标记sr_matching_filter '%s' 没有遵循格式要求"

msgid "Flavor could not be found, skipping migration."
msgstr "没有找到云主机类型，跳过迁移。"

#, python-format
msgid ""
"Found multiple matches for domain %(domain)s.\n"
"%(entry)s"
msgstr ""
"对于域 %(domain)s，找到多个匹配项。\n"
"%(entry)s"

#, python-format
msgid "Get esx cookies failed: %s"
msgstr "获取esx cookies失败：%s"

msgid ""
"Host field should not be set on the instance until resources have been "
"claimed."
msgstr "直到已请求资源，才应该对实例设置主机字段。"

#, python-format
msgid ""
"Hostname %(hostname)s is longer than 63, truncate it to %(truncated_name)s"
msgstr "主机名 %(hostname)s 超过63个字符，缩减为以下字符 %(truncated_name)s"

msgid "Hypervisor driver does not support resume guests"
msgstr "管理程序驱动程序不支持恢复 guest"

#, python-format
msgid "IP %s leased that is not associated"
msgstr "没有关联的IP %s 被租用了"

#, python-format
msgid "IP %s released that is not associated"
msgstr "没有关联的IP %s 被释放了"

#, python-format
msgid "IP %s released that was not leased"
msgstr "没有租用的IP %s 被释放了"

#, python-format
msgid "IP |%s| leased that isn't allocated"
msgstr "没有分配的IP |%s| 被租用了"

#, python-format
msgid "ISO creation tool '%s' does not exist."
msgstr "ISO 创建工具'%s' 不存在。"

#, python-format
msgid ""
"Ignoring DiskNotFound exception while detaching volume %(volume_id)s from "
"%(mp)s: %(err)s"
msgstr "在从%(mp)s断开卷%(volume_id)s的过程中，忽略磁盘没有找到异常：%(err)s"

#, python-format
msgid "Ignoring Volume Error on vol %(vol_id)s during delete %(exc)s"
msgstr "在删除%(exc)s期间，忽略卷%(vol_id)s上的卷错误"

#, python-format
msgid "Ignoring error injecting %(inject)s into image (%(e)s)"
msgstr "正在忽略将 %(inject)s 插入到映像 (%(e)s) 中时发生的错误"

#, python-format
msgid "Ignoring error injecting data into image %(image)s (%(e)s)"
msgstr "忽略注入数据到镜像%(image)s的错误  (%(e)s)"

#, python-format
msgid "Ignoring error while configuring instance with agent: %s"
msgstr "用代理配置实例时，忽略错误：%s"

#, python-format
msgid "Ignoring exception %(exc)s when getting PBDs for %(sr_ref)s"
msgstr "异常 %(exc)s 在为 %(sr_ref)s 得到PBDs时被忽略"

#, python-format
msgid "Ignoring exception %(exc)s when unplugging PBD %(pbd)s"
msgstr "异常 %(exc)s 在拔开PBD %(pbd)s 时被忽略"

#, python-format
msgid "Ignoring malformed capability '%s'. Format should be 'key:val'."
msgstr "忽略不正确的功能'%s'。格式应该是”键：值“  'key:val'。"

#, python-format
msgid ""
"Ignoring supplied device name: %(device_name)s. Libvirt can't honour user-"
"supplied dev names"
msgstr "忽略提供的设备名：%(device_name)s。Libvirt不能使用用户提供的设备名"

#, python-format
msgid "Ignoring supplied device name: %(suggested_dev)s"
msgstr "忽略提供的设备名称：%(suggested_dev)s"

#, python-format
msgid "Ignoring the invalid elements of the option %(name)s: %(options)s"
msgstr "忽略选项  %(name)s: %(options)s 中无效的元素。"

#, python-format
msgid "Ignoring volume cleanup failure due to %s"
msgstr "正在忽略由于 %s 而导致的卷清除故障"

msgid "Image not found during snapshot"
msgstr "快照期间，没有找到镜像"

msgid ""
"In vmwareapi:vmops:_destroy_instance, exception while deleting the VM "
"contents from the disk"
msgstr "在vmwareapi:vmops:_destroy_instance中，从磁盘删除虚拟机内容时出现异常"

#, python-format
msgid ""
"In vmwareapi:vmops:_destroy_instance, got this exception while un-"
"registering the VM: %s"
msgstr ""
"在vmwareapi:vmops:_destroy_instance 中，取消注册虚拟机时得到这个异常：%s"

msgid "Info cache for instance could not be found. Ignore."
msgstr "实例的info 缓存没有找到。忽略。"

#, python-format
msgid ""
"Instance %(instance)s is using a backing file %(backing)s which does not "
"appear in the image service"
msgstr ""
"实例 %(instance)s 正在使用的备份文件 %(backing)s 没有出现在镜像服务里。"

#, python-format
msgid ""
"Instance %(uuid)s appears to not be owned by this host, but by %(host)s. "
"Startup processing is being skipped."
msgstr "实例%(uuid)s 好像不在这台主机上面，而是在主机%(host)s。跳过启动处理。"

msgid "Instance build timed out. Set to error state."
msgstr "实例构建超时。请设置为错误状态。"

msgid "Instance disappeared during reboot"
msgstr "在重新引导期间，实例消失"

msgid "Instance disappeared while detaching a PCI device from it."
msgstr "在从实例解除PCI设备绑定时，实例消失。"

msgid "Instance does not exist on backend"
msgstr "在后台中实例不存在"

msgid ""
"Instance does not exists. Proceeding to delete instance properties on "
"datastore"
msgstr "实例不存在。在数据存储datastore上面删除实例属性。"

#, python-format
msgid "Instance failed network setup (attempt %(attempt)d of %(attempts)d)"
msgstr "实例创建网络失败，进行%(attempts)d)中的(attempt %(attempt)d尝试"

msgid "Instance has had its instance_type removed from the DB"
msgstr "实例已从数据库中除去其 instance_type"

#, python-format
msgid ""
"Instance in transitional state (%(task_state)s) at start-up and power state "
"is (%(power_state)s), clearing task state"
msgstr ""
"开始时实例在变迁状态(%(task_state)s)，电源状态是 (%(power_state)s)，清除任务"
"状态"

#, python-format
msgid ""
"Instance in transitional state (%(task_state)s) at start-up and power state "
"is (%(power_state)s), clearing task state and unpausing the instance"
msgstr ""
"开始时实例在变迁状态(%(task_state)s)，电源状态是 (%(power_state)s)，清除任务"
"状态并且取消暂停实例"

msgid "Instance is not (soft-)deleted."
msgstr "实例未（软）删除。"

#, python-format
msgid ""
"Instance is not stopped. Calling the stop API. Current vm_state: "
"%(vm_state)s, current task_state: %(task_state)s, original DB power_state: "
"%(db_power_state)s, current VM power_state: %(vm_power_state)s"
msgstr ""
"实例不是停止状态。调用stop API。当前虚拟机状态：%(vm_state)s，当前任务状态："
"%(task_state)s，原始DB 电源状态：%(db_power_state)s，当前虚拟机电源状态："
"%(vm_power_state)s"

msgid "Instance is paused unexpectedly. Ignore."
msgstr "实例已意外暂停。请忽略。"

msgid "Instance is suspended unexpectedly. Calling the stop API."
msgstr "实例已意外暂挂。正在调用“停止 API”。"

msgid "Instance is unexpectedly not found. Ignore."
msgstr "意外找不到实例。请忽略。"

msgid ""
"Instance lifecycle events from the compute driver have been disabled. Note "
"that lifecycle changes to an instance outside of the compute service will "
"not be synchronized automatically since the _sync_power_states periodic task "
"is also disabled."
msgstr ""
"从计算驱动中实例生命周期事件被禁用。注意，从计算服务以外进行生命周期的的改变"
"将不会被自动同步，因为_sync_power_states 周期性任务也同样被禁用。"

msgid "Instance may be still running, destroy it again."
msgstr "实例可能仍在运行，请再次将其销毁。"

msgid "Instance not resizing, skipping migration."
msgstr "未在调整实例大小，正在跳过迁移。"

#, python-format
msgid ""
"Instance shutdown by itself. Calling the stop API. Current vm_state: "
"%(vm_state)s, current task_state: %(task_state)s, original DB power_state: "
"%(db_power_state)s, current VM power_state: %(vm_power_state)s"
msgstr ""
"实例被它自己关闭。调用stop API。当前虚拟机虚拟机状态： %(vm_state)s，当前任务"
"状态：%(task_state)s，原始DB 电源状态： %(db_power_state)s，当前VM 电源状态："
"%(vm_power_state)s"

#, python-format
msgid ""
"Instance uuid could not be retrieved for instance %s. Instance state change "
"event will be ignored."
msgstr "不能为实例%s 获取实例uuid。忽略实例状态变化事件。"

msgid "Invalid 'agent_present' value. Falling back to the default."
msgstr "无效 'agent_present' 值。回归到缺省。"

#, python-format
msgid "Invalid cachemode %(cache_mode)s specified for disk type %(disk_type)s."
msgstr "为磁盘类型 %(disk_type)s 指定的高速缓存方式 %(cache_mode)s 无效。"

#, python-format
msgid "Invalid value '%d' for image_compression_level"
msgstr "无效镜像压缩级别 image_compression_level的值'%d'"

#, python-format
msgid "Invalid value '%s' for torrent_images"
msgstr "无效种子镜像 torrent_images的值'%s' "

#, python-format
msgid "Invalid value for id mapping %s"
msgstr "id 映射%s 的值无效"

#, python-format
msgid "Key '%(key)s' not in instances '%(inst)s'"
msgstr "键“%(key)s”未在实例“%(inst)s”中"

msgid ""
"Libvirt module could not be loaded. NWFilterFirewall will not work correctly."
msgstr "Libvirt模块无法加载。NWFilterFirewall 无法正常工作。"

#, python-format
msgid "Live migration not completed after %d sec"
msgstr "在%d秒后，在线迁移不能完成"

#, python-format
msgid "Live migration stuck for %d sec"
msgstr "在线迁移卡住 %d 秒"

#, python-format
msgid ""
"Lost connection to XenAPI during call to %(plugin)s.%(fn)s.  Retrying call."
msgstr "调用%(plugin)s.%(fn)s 期间，丢失到XenAPI的连接。重试调用。"

msgid "Lost connection to memcache server for reporting service status."
msgstr "丢失到 memcache 服务器的连接，连接用来报告服务状态。"

msgid "Lost connection to nova-conductor for reporting service status."
msgstr "丢失到nova-conductor连接，连接用来报告服务状态。"

#, python-format
msgid ""
"Metadata value %(value)s for %(key)s is not of type %(type)s. Using default "
"value %(default)s."
msgstr ""
"%(key)s 的元数据值 %(value)s 不是类型 %(type)s。使用缺省值 %(default)s。"

#, python-format
msgid "Migration %s is not found."
msgstr "没有找到迁移 %s "

msgid "Migration operation was cancelled"
msgstr "迁移操作取消"

#, python-format
msgid "Mountpoint cannot be translated: %s"
msgstr "挂载点无法被翻译：%s"

#, python-format
msgid "Multiple base images for image: %s"
msgstr "多个镜像%s的基本镜像"

#, python-format
msgid ""
"Network %(id)s not matched with the tenants network! The ports tenant "
"%(tenant_id)s will be used."
msgstr "网络 %(id)s 与租户网络不匹配！将使用端口租户 %(tenant_id)s 。"

#, python-format
msgid "Neutron error: Fixed IP %s is already in use."
msgstr "Neutorn错误：固定IP %s 已经在使用。"

#, python-format
msgid ""
"Neutron error: MAC address %(mac)s is already in use on network %(network)s."
msgstr "Neutron 错误：MAC 地址%(mac)s 已经在网络%(network)s 上使用。"

#, python-format
msgid "Neutron error: No more fixed IPs in network: %s"
msgstr "Neutron 错误：网络：%s 中没有更多固定IP"

#, python-format
msgid "Neutron error: Port quota exceeded in tenant: %s"
msgstr "Neutron 错误：租户 %s 中的端口配额超过"

#, python-format
msgid "No Network for FixedIP: %s"
msgstr "固定IP：%s 没有网络"

#, python-format
msgid "No VirtualInterface for FixedIP: %s"
msgstr "固定IP：%s 没有虚拟接口"

#, python-format
msgid "No cell_name for %(method)s() from API"
msgstr "API 的%(method)s() 方法中没有cell_name"

msgid "No cell_name for instance update from API"
msgstr "API没有实例更新的cell_name"

#, python-format
msgid "No compute node record for %(host)s:%(node)s"
msgstr "没有%(host)s:%(node)s 的计算节点记录"

#, python-format
msgid "No compute service record found for host %(host)s"
msgstr "没有找到主机 %(host)s 可用的计算服务记录"

msgid "No free nbd devices"
msgstr "没有空闲NBD设备"

#, python-format
msgid ""
"No host name specified for the notification of HostAPI.%s and it will be "
"ignored"
msgstr "没有为HostAPI 的通知指定的主机名。%s 将会被忽略"

#, python-format
msgid "No match when trying to update BDM: %(bdm)s"
msgstr "尝试更新BDM：%(bdm)s时，没有匹配内容"

msgid "No more network or fixed IP to be allocated"
msgstr "没有多余的网络或fixed IP可被分配"

msgid "No valid host found for rebuild"
msgstr "重建过程中没有找到有效主机"

msgid "No valid host found for unshelve instance"
msgstr "没有找未停止实例的有效主机"

#, python-format
msgid ""
"Node %(uuid)s has a malformed \"%(prop)s\". It should be an integer but its "
"value is \"%(value)s\"."
msgstr ""
"节点 %(uuid)s 有一个不正确的属性\"%(prop)s\"。它应该是一个整数，但是它的值是 "
"\"%(value)s\"。"

#, python-format
msgid "Node %(uuid)s has a malformed \"%(prop)s\". It should be an integer."
msgstr "节点 %(uuid)s 有一个不正确的属性\"%(prop)s\"。它应该是一个整数。"

msgid ""
"Node field should not be set on the instance until resources have been "
"claimed."
msgstr "直到已请求资源，才应该对实例设置主机字段.。"

#, python-format
msgid "Not deleting key %s"
msgstr "不删除秘钥 %s"

msgid ""
"Old property name \"hw_watchdog_action\" is now deprecated and will be "
"removed in the next release. Use updated property name \"hw:watchdog_action"
"\" instead"
msgstr ""
"现在旧属性名\"hw_watchdog_action\" 被废弃了，将在下一个版本中移除。使用更新过"
"的属性 \"hw:watchdog_action\" 作为代替。"

msgid "Paused instance shutdown by itself. Calling the stop API."
msgstr "暂停的实例自己关闭。调用stop API。"

#, python-format
msgid "Periodic cleanup failed to delete instance: %s"
msgstr "周期性清除，删除实例失败：%s"

#, python-format
msgid "Periodic reclaim failed to delete instance: %s"
msgstr "周期性重新声明，删除实例失败：%s"

#, python-format
msgid ""
"Periodic task is updating the host stat, it is trying to get disk "
"%(i_name)s, but access is denied. It is most likely due to a VM that exists "
"on the compute node but is not managed by Nova."
msgstr ""
"周期性任务更新主机统计，尝试获取磁盘%(i_name)s，但是拒绝访问。最大可能是在计"
"算主机上面存在一个不被Nova管理的虚拟机VM。"

#, python-format
msgid ""
"Periodic task is updating the host stat, it is trying to get disk "
"%(i_name)s, but disk file was removed by concurrent operations such as "
"resize."
msgstr ""
"周期性任务更新这台主机统计。尝试获取磁盘%(i_name)s，但是磁盘文件被并发操作移"
"除，比如调整大小resize。"

#, python-format
msgid ""
"Periodic task is updating the host stats, it is trying to get disk info for "
"%(i_name)s, but the backing volume block device was removed by concurrent "
"operations such as resize. Error: %(error)s"
msgstr ""
"周期性任务更新主机统计，尝试获取磁盘%(i_name)s信息，但是备份卷块设备被并发操"
"作移除，比如调整大小resize。错误：%(error)s"

msgid ""
"Please consider to execute setup before trying to configure debug log "
"message."
msgstr "在尝试配置调试debug 日志消息之前，请考虑执行创建。"

#, python-format
msgid "Port %s does not exist"
msgstr "端口号 %s  不存在。"

#, python-format
msgid "Power state %s not found."
msgstr "没有找到电源状态 %s 。"

#, python-format
msgid "Quota exceeded for %s, tried to allocate floating IP"
msgstr "超过%s 的配额，尝试分配浮动IP"

#, python-format
msgid ""
"Quota exceeded for project %(pid)s, tried to allocate fixed IP. %(used)s of "
"%(allowed)s are in use or are already reserved."
msgstr ""
"项目%(pid)s 超过配额，尝试分配固定IP。允许的%(allowed)s 中的%(used)s 已在使"
"用 或者已经保留。"

msgid "Race detected in libvirt.virDomain.info, trying one more time"
msgstr "libvirt.virDomain.info 中检测到环Race ，再尝试一次。"

msgid "Reboot failed but instance is running"
msgstr "重启失败，但是实例仍然运行"

msgid ""
"Reboot failed due to bad volumes, detaching bad volumes and starting halted "
"instance"
msgstr ""
"由于以下原因，重新引导失败：卷不正确，正在与不正确的卷断开连接，以及正在启动"
"已停止的实例"

msgid "Rejecting snapshot request, snapshots currently disabled"
msgstr "拒绝快照请求，快照当前未被激活"

msgid ""
"Relative blockrebase support was not detected. Continuing with old behaviour."
msgstr "相对块重新基线blockrebase支持没有检测到。以老的方式继续。"

#, python-format
msgid ""
"Report interval must be less than service down time. Current config: "
"<service_down_time: %(service_down_time)s, report_interval: "
"%(report_interval)s>. Setting service_down_time to: %(new_service_down_time)s"
msgstr ""
"报告间隔必须小于服务宕机时间service_down_time。当前配置：<service_down_time: "
"%(service_down_time)s, report_interval: %(report_interval)s>. Setting "
"service_down_time to: %(new_service_down_time)s"

#, python-format
msgid "Retry SR scan due to error: %s"
msgstr "重试SR 扫描，原因是错误：%s"

#, python-format
msgid ""
"Retrying %(name)s after a MessagingTimeout, attempt %(attempt)s of "
"%(retries)s."
msgstr "消息超时后，重试 %(name)s，尝试 %(retries)s 的%(attempt)s 次。"

#, python-format
msgid "Root disk file creation failed - %s"
msgstr "根磁盘创建失败- %s"

#, python-format
msgid ""
"Running Nova with a VMware vCenter version less than %(version)s is "
"deprecated. The required minimum version of vCenter will be raised to "
"%(version)s in the 13.0.0 release."
msgstr ""
"Nova不再支持低于%(version)s 的VMware vCenter 。在13.0.0发布版本中，最小"
"VCenter版本是%(version)s。"

msgid ""
"Running libvirt-lxc without user namespaces is dangerous. Containers spawned "
"by Nova will be run as the host's root user. It is highly suggested that "
"user namespaces be used in a public or multi-tenant environment."
msgstr ""
"不再用户命名空间运行libvirt-lxc是非常危险的。Nova生产的容器将以主机root用户的"
"身份运行。强烈建议在一个公共或多租户空间中使用用户命名空间。"

#, python-format
msgid ""
"Selected host: %(host)s failed to consume from instance. Error: %(error)s"
msgstr "选择的主机：%(host)s 从实例消费失败。错误：%(error)s"

msgid "Server with unsupported policy cannot be rebuilt"
msgstr "有不支持策略的服务器不能被重建。"

msgid "Service killed that has no database entry"
msgstr "因无数据库记录，服务已被中止"

#, python-format
msgid "Setting instance to %s state."
msgstr "设置实例到 %s 状态。"

#, python-format
msgid "Setting migration %(migration_id)s to error: %(reason)s"
msgstr "正在将迁移 %(migration_id)s 设置为“错误”：%(reason)s"

#, python-format
msgid "Skipping detach because VBD for %s was not found"
msgstr "跳过绑定，因为没有找到%s 的VBD"

#, python-format
msgid ""
"Subnet(s) too large, defaulting to /%s.  To override, specify network_size "
"flag."
msgstr "子网太大，缺省为 /%s。要进行覆盖，请指定 network_size 标记。"

#, python-format
msgid ""
"Tenant_id %(tenant_id)s does not match tenant_id of instance %(instance_id)s."
msgstr "Tenant_id %(tenant_id)s 不匹配实例的 tenant_id %(instance_id)s."

msgid ""
"The cells feature of Nova is considered experimental by the OpenStack "
"project because it receives much less testing than the rest of Nova. This "
"may change in the future, but current deployers should be aware that the use "
"of it in production right now may be risky. Also note that cells does not "
"currently support rolling upgrades, it is assumed that cells deployments are "
"upgraded lockstep so n-1 cells compatibility does not work."
msgstr ""
"OpenStack项目Nova的单元特性做为实验性的特性，因为它比Nova的其它模块接受了更少"
"的测试。在将来将会发生改变，但是当前部署者应该保持谨慎，立即把它用到生产环境"
"是有风险的。同时注意到单元当前不支持回归更新，假设单元部署者升级中出现失败步"
"骤，那么n-1个单元兼容性将不能正常工作。"

#, python-format
msgid ""
"The same keypair name '%(name)s' exists in the following cells: "
"%(cell_names)s. The keypair value from the first cell is returned."
msgstr ""
"在以下单元: %(cell_names)s中存在相同的密钥对名字  '%(name)s' 。返回第一个单元"
"的密钥对值。"

msgid ""
"This key manager is insecure and is not recommended for production "
"deployments"
msgstr "秘钥管理器是不安全的，不推荐在生产部署环境中使用"

msgid "This shouldn't be getting called except during testing."
msgstr "不应该调用此项（在测试期间除外）。"

msgid ""
"Timed out waiting for nova-conductor.  Is it running? Or did this service "
"start before nova-conductor?  Reattempting establishment of nova-conductor "
"connection..."
msgstr ""
"等待nova-conductor 超时。它在运行吗？或者这个服务在nova-conductor之前启动？尝"
"试重新建立nova-conductor连接。。。"

msgid "Timed out while waiting for soft shutdown."
msgstr "等待软关机时，超时。"

#, python-format
msgid "Timeout waiting for vif plugging callback for instance %(uuid)s"
msgstr "等待实例%(uuid)s的vif 插入回调超时"

msgid "Too many id maps, only included first five."
msgstr "太多id映射，只能包含前5个。"

#, python-format
msgid ""
"Treating negative config value (%(retries)s) for 'block_device_retries' as 0."
msgstr " 把'block_device_retries' 负数的配置值 (%(retries)s) 当做0。"

#, python-format
msgid ""
"Treating negative config value (%(retries)s) for 'glance.num_retries' as 0."
msgstr "把'glance.num_retries' 的负数的配置值 (%(retries)s) 当做0。"

#, python-format
msgid ""
"Tried to remove rule that was not there: %(chain)r %(rule)r %(wrap)r %(top)r"
msgstr "试图移除不存在的规则：%(chain)r %(rule)r %(wrap)r %(top)r"

#, python-format
msgid ""
"Trying to remove device with %(status)s ownership %(instance_uuid)s because "
"of %(pci_exception)s"
msgstr ""
"由于%(pci_exception)s，尝试移除%(status)s 的设备，设备所有者"
"是%(instance_uuid)s"

#, python-format
msgid "URI %(uri)s does not support connection events: %(error)s"
msgstr "URI %(uri)s 不支持连接事件：%(error)s"

#, python-format
msgid "URI %(uri)s does not support events: %(error)s"
msgstr "URI %(uri)s 不支持事件：%(error)s"

#, python-format
msgid "URI %(uri)s does not support full set of host capabilities: %(error)s"
msgstr "URI %(uri)s 不支持主机能力的全部集合：%(error)s"

#, python-format
msgid "URI %(uri)s does not support listDevices: %(error)s"
msgstr "URI %(uri)s 不支持listDevices：%(error)s"

#, python-format
msgid "Unable to delete %(file)s. Exception: %(ex)s"
msgstr "不能删除%(file)s。异常：%(ex)s"

#, python-format
msgid "Unable to dequalify.  %(name)s is not in %(domain)s.\n"
msgstr "无法取消限定。%(name)s 没有在 %(domain)s 中。\n"

msgid ""
"Unable to find a different image to use for rescue VM, using instance's "
"current image"
msgstr "找不到要用于安全 VM 的另一映像，正在使用实例的当前映像"

#, python-format
msgid ""
"Unable to find network matching '%(network_name)s', user will have to enter "
"IP manually..."
msgstr "不能找到匹配 '%(network_name)s'的网络，用户必须手动输入IP。。。"

#, python-format
msgid "Unable to force TCG mode, libguestfs too old? %s"
msgstr "不能强制TCG 模式，libguestfs 太旧吗？%s"

#, python-format
msgid "Unable to increase max downtime to %(time)dms: %(e)s"
msgstr "不能减少最大宕机时间到 %(time)dms：%(e)s"

#, python-format
msgid "Unable to mount image %(image)s with error %(error)s. Cannot resize."
msgstr "不能挂着镜像 %(image)s ，镜像有错误%(error)s。不能重新调整大小。"

#, python-format
msgid "Unable to stat %(disk)s: %(ex)s"
msgstr "不能统计%(disk)s：%(ex)s"

msgid "Unable to update metadata, VM not found."
msgstr "不能更新元数据，虚拟机没有找到。"

#, python-format
msgid "Unable to update the agent due to: %(exc)s"
msgstr "不能更新代理，原因是：%(exc)s"

#, python-format
msgid ""
"Unexpected confirmation status '%(status)s' of migration %(id)s, exit "
"confirmation process"
msgstr "未预期的迁移%(id)s的确认状态'%(status)s'，退出确认过程"

#, python-format
msgid "Unexpected migration job type: %d"
msgstr "未预期的迁移工作类型： %d"

#, python-format
msgid "Unexpected power state %d"
msgstr "电源处于意外状态 %d"

#, python-format
msgid "Unknown base file: %s"
msgstr "未知的基文件：%s"

#, python-format
msgid "Unknown content in connection_info/qos_specs: %s"
msgstr "在 connection_info/qos_specs中，未知内容：%s"

#, python-format
msgid "Unrecognized cache_images value '%s', defaulting to True"
msgstr "无法识别 cache_images 值“%s”，缺省为 True"

msgid "VCPUs not set; assuming CPU collection broken"
msgstr "未设置 VCPU 数；假定 CPU 收集已中断"

msgid "VM already halted, skipping shutdown..."
msgstr "VM 已停止，正在跳过关闭..."

msgid "VM is not present, skipping destroy..."
msgstr "虚拟机不存在，跳过销毁..."

msgid "VM is not present, skipping soft delete..."
msgstr "VM 不存在，正在跳过软删除..."

msgid ""
"Value specified in conf file for xenserver.torrent_base_url does not contain "
"a slash character, therefore it will not be used as part of the torrent URL. "
"Specify a valid base URL as defined by RFC 1808 (see step 6)."
msgstr ""
"配置文件中指定的xenserver.torrent_base_url 的值没有包含斜线/字符，因而它将不"
"会作为种子URL的一部分。指定一个有效的RFC 1808 定义的基本URL（参考第6步）。"

#, python-format
msgid "Virtual disk: %s of volume's backing not found."
msgstr "没有找到卷的备份的虚拟磁盘 %s。"

#, python-format
msgid "Volume %s should be encrypted but there is no encryption metadata."
msgstr "卷 %s 应该被加密，但是没有加密元数据。"

#, python-format
msgid ""
"Volume id: %(vol_id)s finished being created but its status is "
"%(vol_status)s."
msgstr "卷id：%(vol_id)s 完成了创建，但是它的状态是%(vol_status)s。"

msgid ""
"WARNING: Using the remote debug option changes how Nova uses the eventlet "
"library to support async IO. This could result in failures that do not occur "
"under normal operation. Use at your own risk."
msgstr ""
"警告：使用远程debug选项会改变Nova调用携程库对异步IO的使用。 这可能导致普通模"
"式下不会出现的错误。 请考虑使用风险。"

#, python-format
msgid "Warning, memory usage is 0 for %(instance)s on baremetal node %(node)s."
msgstr "警告，在裸机 %(node)s 上的实例%(instance)s 内存使用率是0。"

#, python-format
msgid ""
"Warning, number of cpus is 0 for %(instance)s on baremetal node %(node)s."
msgstr "警告，在裸机 %(node)s 上的实例%(instance)s cpu个数是0。"

#, python-format
msgid ""
"While synchronizing instance power states, found %(num_db_instances)s "
"instances in the database and %(num_vm_instances)s instances on the "
"hypervisor."
msgstr ""
"当同步实例电源状态时，发现数据中实例数 %(num_db_instances)s 和监测器上的实例"
"数%(num_vm_instances)s"

msgid ""
"X-Instance-ID present in request headers. The 'service_metadata_proxy' "
"option must be enabled to process this header."
msgstr ""
"请求头中有X-Instance-ID。必须启用'service_metadata_proxy' 选项来处理这头文"
"件。"

#, python-format
msgid ""
"X-Instance-ID-Signature: %(signature)s does not match the expected value: "
"%(expected_signature)s for id: %(requestor_id)s. Request From: "
"%(requestor_address)s"
msgstr ""
"X-Instance-ID-Signature: %(signature)s 不符合预期值：id %(requestor_id)s "
"的%(expected_signature)s 。请求来源： %(requestor_address)s"

#, python-format
msgid ""
"You are running with libvirt version %s which is known to have broken NUMA "
"support. Consider patching or updating libvirt on this host if you need NUMA "
"support."
msgstr ""
"您正在运行的libvirt 版本 %s，该版本已知对NUMA支持存在漏洞。如果您需要NUMA支"
"持，请在主机上打补丁或升级libvirt。"

#, python-format
msgid "_migrate_disk_resizing_up failed to rollback: %s"
msgstr "_migrate_disk_resizing_up 回归失败：%s"

#, python-format
msgid ""
"attach interface failed , try to deallocate port %(port_id)s, reason: %(msg)s"
msgstr "绑定接口失败，尝试取消分配端口%(port_id)s，原因是：%(msg)s"

#, python-format
msgid "couldn't obtain the memory from domain: %(uuid)s, exception: %(ex)s"
msgstr "不能从域：%(uuid)s 获取内存，异常： %(ex)s"

#, python-format
msgid "cpu_arch not defined for node '%s'"
msgstr "没有为节点'%s'定义cpu 架构cpu_arch"

msgid "datastore_regex is ignored when PBM is enabled"
msgstr "当PBM 启用时，忽略数据存储正则表达式datastore_regex"

#, python-format
msgid "deallocate port %(port_id)s failed"
msgstr "取消分配端口%(port_id)s失败"

#, python-format
msgid "hostname %s too long, truncating."
msgstr "主机名%s 太长，缩短。"

#, python-format
msgid "ignoring missing logical volume %(path)s"
msgstr "忽略丢失的逻辑卷 %(path)s "

#, python-format
msgid ""
"image %(id)s at (%(base_file)s): warning -- an absent base file is in use! "
"instances: %(instance_list)s"
msgstr ""
"(%(base_file)s) 处的映像 %(id)s：警告 - 正在使用不存在的基本文件！实例："
"%(instance_list)s"

#, python-format
msgid "image %(volume)s in pool %(pool)s can not be found, failed to remove"
msgstr "不能发现池 %(pool)s 中的卷 %(volume)s，移除失败"

#, python-format
msgid "instance's host %s is down, deleting from database"
msgstr "实例的主机 %s 处于关闭状态，正在从数据库进行删除"

#, python-format
msgid ""
"instance-dns-zone is |%(domain)s|, which is in availability zone |%(zone)s|. "
"Instance is in zone |%(zone2)s|. No DNS record will be created."
msgstr ""
"instance-dns-zone 是 |%(domain)s|，它在可用性区域 |%(zone)s| 中。实例在区域 |"
"%(zone2)s| 中。将不创建任何 DNS 记录。"

#, python-format
msgid "instance-dns-zone not found |%s|."
msgstr "没有发现instance-dns-zone %s|。"

msgid "ipxe_boot is True but no ISO image found"
msgstr "IPXE_boot为真True ，但是没有找到ISO镜像"

msgid "ipxe_boot_menu_url not set, user will have to enter URL manually..."
msgstr "没有设置ipxe_boot_menu_ur，用户必须手动输入URL。。。"

msgid "ipxe_network_name not set, user will have to enter IP manually..."
msgstr "没有设置ipxe_network_name，用户必须手动输入IP。。。"

#, python-format
msgid "isLuks exited abnormally (status %(exit_code)s): %(stderr)s"
msgstr "isLuks 非正常退出(状态 %(exit_code)s): %(stderr)s"

#, python-format
msgid "multiple fixed_ips exist, using the first IPv4 fixed_ip: %s"
msgstr "存在多个固定ips，使用第一个IPv4 固定ip：%s"

#, python-format
msgid ""
"my_ip address (%(my_ip)s) was not found on any of the interfaces: %(ifaces)s"
msgstr "在任何接口：%(ifaces)s上，my_ip 地址 (%(my_ip)s) 没有找到。"

#, python-format
msgid "plug_vifs() failed %(cnt)d. Retry up to %(max_retry)d."
msgstr "plug_vifs() 失败 %(cnt)d。重试到%(max_retry)d。"

#, python-format
msgid "rbd remove %(volume)s in pool %(pool)s failed"
msgstr "rbd 移除池 %(pool)s 的卷   %(volume)s 失败"

msgid "set_admin_password is not implemented by this driver or guest instance."
msgstr "此驱动程序或 guest 实例未实现 set_admin_password。"

#, python-format
msgid ""
"trying to inject a file into a non-running (state: %(current_state)s "
"expected: %(expected_state)s)"
msgstr ""
"尝试注入文件到一个非运行态的实， (状态: %(current_state)s 期望: "
"%(expected_state)s)"

#, python-format
msgid ""
"trying to reboot a non-running instance: (state: %(state)s expected: "
"%(running)s)"
msgstr ""
"正在尝试重新引导未在运行的实例：（状态：%(state)s，期望的是：%(running)s）"

#, python-format
msgid ""
"trying to snapshot a non-running instance: (state: %(state)s expected: "
"%(running)s)"
msgstr ""
"正在尝试对未在运行的实例生成快照：（状态：%(state)s，期望的是：%(running)s）"

#, python-format
msgid ""
"vmware_ostype from image is not recognised: '%(ostype)s'. An invalid os type "
"may be one cause of this instance creation failure"
msgstr ""
"镜像中的vmware_ostype没有被识别： '%(ostype)s'。无效os类型可能是导致这台实例"
"创建失败的一个原因。"
